pocs/scheduler/dispatch.py

```diff
@@ -53,15 +53,15 @@ def get_observation(self, time=None, show_all=False, reread_fields_file=False):
         }
 
         for constraint in listify(self.constraints):
-            self.logger.debug("Checking Constraint: {}".format(constraint))
+            self.logger.info("Checking Constraint: {}".format(constraint))
             for obs_name, observation in self.observations.items():
                 if obs_name in valid_obs:
                     self.logger.debug("\tObservation: {}".format(obs_name))
 
                     veto, score = constraint.get_score(
                         time, self.observer, observation, **common_properties)
 
-                    self.logger.debug("\t\tScore: {}\tVeto: {}".format(score, veto))
+                    self.logger.debug("\t\tScore: {:.05f}\tVeto: {}".format(score, veto))
```

Comment:
Again with the leading tabs. I suppose it makes it easier to find... sometimes...